doc/api/globals.md

```diff
@@ -29,27 +29,20 @@ added: v0.1.27
 
 * {String}
 
-The name of the directory that the currently executing script resides in.
+The directory name of the current module. This the same as the
+[`path.dirname()`][] of the [`__filename`][].
+
+`__dirname` isn't actually a global but rather local to each module.
```

Comment:
Yes, reads better, but if I write it like that its hard to get __filename linked, I'll try to figure out a way.